src/scaffolding/wizard/java/JavaGatherInformationStep.ts

```diff
@@ -0,0 +1,80 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as fse from 'fs-extra';
+import * as gradleParser from 'gradle-to-js/lib/parser';

```

Comment:
Just FYI-RANT kind of remark: my experience with Gradle is that it is very much a moving target, and they do not hesitate to introduce many breaking changes in every version. On top of that they are trying to move away from the old "Groovy" syntax to the new "Kotlin" syntax and deprecate the Groovy syntax--which is the only thing that `gradle-to-js` parser supports.

But what we have is better than nothing, and with this change it will be certainly easier for people outside the core team to submit patches!

Reply:
Given the 3+ ways of getting the JAR name that this code (and the old code) account for, it is clear you are correct :smile: